.travis.yml

```diff
@@ -8,19 +8,20 @@ go:
 services:
   - docker
 
-go_import_path: gopkg.in/bblfsh/sdk.v2
+go_import_path: github.com/bblfsh/sdk
 
 matrix:
   fast_finish: true
   allow_failures:
     - go: tip
 
+env:
+  - GO111MODULE=on
 
 before_install:
-  - cd $GOPATH/src/gopkg.in/bblfsh/sdk.v2
+  - cd $GOPATH/src/github.com/bblfsh/sdk
   - make validate-commit
-  - wget -O dep https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64 && chmod +x dep
-  - ./dep ensure --vendor-only
+  - go mod download
```

Comment:
a bit hard to understand how valuable is to download that before build/test  (e.g could be very valuable if it's warming up a cache for multiple runs but not sure if that is the case)

Reply:
For now, it only separates the dependency download process from the test, nothing more. But if I'm not mistaken, there is a way to cache data across builds. So later we can add Go modules cache to it, for example.